### DISCUSSION TITLE
Feature Request: Allow only maintainers the ability to approve PRs

### DISCUSSION THREAD

**Discussion by ide** [community/community#23959] [New to GitHub]
<p>On occasion I’ve seen PRs marked as “approved” by people who aren’t the maintainers of the repo at hand and find it noisy at best and possibly misleading. The maintainers of the repo are almost always the ones who are best equipped to review a PR and bear the responsibility of supporting it down the road and approval from a non-maintainer doesn’t indicate whether the maintainer will merge the PR. Approvals from maintainers are much more salient.</p>
<p>I suspect “approvals” from non-maintainers often come from people who want the PR to be merged for their own reasons, either because they want the same feature or bug fixed and are possibly co-workers of the PR authors and are signaling the PR works fine for them and they want it ASAP. There are other mechanisms for being helpful (ex: providing a test plan) or signaling interest (ex: :+1: reactions) that are more helpful and communicate intent better.</p>
<p>This feature might not need to be an option or checkbox and could be the default across the board. Approvals from non-collaborators aren’t net positive in my experience and restricting them to collaborators only would keep it clearer whether a PR is actually likely to be merged or not.</p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hi <a class="mention" href="/ide">@ide</a>,</p>
<p>Thanks for this feedback! I’ve logged your feature request in our internal feature request list. Though I can’t guarantee anything or share a timeline for this, I can tell you that it’s been shared with the appropriate teams for consideration.</p>
<p>Please let me know if you have any other questions.</p>
<p>Cheers!</p>

**Comment by ggirotto** [comment]
<p>Is there an update in this issue? I can see some repositories as <code>github.com/flutter/flutter</code> as an example, where only maintainers can approve PRs. How do they achieve this settings?</p>
